setup.py

```diff
@@ -10,7 +10,7 @@
 from distutils.spawn import find_executable
 from distutils import log
 from platform import architecture
-from subprocess import Popen, CalledProcessError, PIPE, check_call
+from subprocess import Popen, CalledProcessError, PIPE, check_call, check_output
```

Comment:
It's the exact same thing 😄. Atleast it sounds like `_check_output` was copied from py27 so that earlier version (py26) could use it. I'm reviewing the changes we can make after dropping old python versions and noticed your code.